client/landing/stepper/declarative-flow/internals/steps-repository/playground/lib/resolve-blueprint-from-url.ts

```diff
@@ -105,11 +105,9 @@ export async function resolveBlueprintFromURL( url: URL ) {
 	blueprint.preferredVersions!.wp =
 		query.get( 'wp' ) || blueprint.preferredVersions!.wp || 'latest';
 
-	// Features, must have networking enabled.
+	// Features
```

Comment:
Let's change this comment to something more descriptive.

Reply:
This is a file copied over from playground package, I would rather not touch it without modifying it at source. That has been my intention, with the change I reverted as well.

Reply:
Ok!